docs/content/installation/installation-with-helm.md

```diff
@@ -249,7 +249,7 @@ The following tables lists the configurable parameters of the NGINX Ingress Cont
 |``controller.podDisruptionBudget.minAvailable`` | The number of Ingress Controller pods that should be available. | 0 |
 |``controller.podDisruptionBudget.maxUnavailable`` | The number of Ingress Controller pods that can be unavailable. | 0 |
 |``controller.strategy`` | Specifies the strategy used to replace old Pods with new ones. Docs for [Deployment update strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) and [Daemonset update strategy](https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/#daemonset-update-strategy). | {} |
-| `controller.disableIPV6` | Disable IPV6 listeners explicitly for nodes that do not support the IPV6 stack. | false |
+|``controller.disableIPV6`` | Disable IPV6 listeners explicitly for nodes that do not support the IPV6 stack. | false |
```

Comment:
I am guessing this fix is being snuck in?

Reply:
Not a fan of sneaking. It may have stowed away on the PR inadvertently.

Reply:
whoops, yes this was staged on different docs branch. Will undo 👍🏼